src/constraints/consumer.jl

```diff
@@ -19,7 +19,7 @@ function add_consumer_constraints!(model, constraints, graph, sets)
 
     # - Balance constraint (using the lowest temporal resolution)
     df = filter(:asset => ∈(Ac), constraints[:highest_in_out].indices; view = true)
-    model[:consumer_balance] = [
+    return model[:consumer_balance] = [
```

Comment:
I wonder if this is correct or these should all be `return nothing`.

Reply:
The JuliaFormatter is technically correct, because these were indeed being returned implicitly. But we don't expect any return, so adding an empty `return` after this is better, in my opinion